examples/pxScene2d/src/pxScene2d.h

```diff
@@ -899,6 +905,10 @@ class pxScene2d: public rtObject, public pxIView, public rtIServiceProvider
   rtMethod1ArgAndNoReturn("addServiceProvider", addServiceProvider, rtFunctionRef);
   rtMethod1ArgAndNoReturn("removeServiceProvider", removeServiceProvider, rtFunctionRef);
 
+#ifdef PXSCENE_SUPPORT_STORAGE
+  rtReadOnlyProperty(storage,storage,rtObjectRef);
```

Comment:
Let's expose this property always but have it return RT_FAIL if storage is compiled off.